setup.cfg

```diff
@@ -55,6 +55,93 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+performance =
+    bottleneck>=1.3.2
+    numba>=0.53.0
+    numexpr>=2.7.1
+computation =
+    numba>=0.53.0
```

Comment:
I would remove `numba` from `computation` since it's in performance. `numba`'s use is only for opt-in performance